BZ-1857151: Added a paragraph about shared VPC with link (#32265)
openshift/openshift-docs: installing/installing_gcp/installing-gcp-user-infra-vpc.adoc

toc::[]

In {product-title} version {product-version}, you can install a cluster into a shared Virtual Private Cloud (VPC) on Google Cloud Platform (GCP) that uses infrastructure that you provide. In this context, a cluster installed into a shared VPC is a cluster that is configured to use a VPC from a project different from where the cluster is being deployed. For more information, see link:https://cloud.google.com/vpc/docs/shared-vpc#shared_vpc_networks[shared VPC networks] in the GCP documentation.

> 

Reviewer: shared VPC networks is a section of Shared VPC overview. So I would suggest to keep one link. I prefer to keep the Shared VPC overview link. We can remove "For more information, see link:https://cloud.google.com/vpc/docs/shared-vpc#shared_vpc_networks[shared VPC networks] in the GCP documentation."

> A shared VPC enables an organization to connect resources from multiple projects to a common VPC network. You can communicate within the organization securely and efficiently by using internal IPs from that network. For more information about VPC, see link:https://cloud.google.com/vpc/docs/shared-vpc[Shared VPC overview] in the GCP documentation.

Reviewer: For more information about VPC - It's supposed to be "For more information about **Shared** VPC".

In {product-title} version {product-version}, you can install a cluster into a shared Virtual Private Cloud (VPC) on Google Cloud Platform (GCP) that uses infrastructure that you provide. In this context, a cluster installed into a shared VPC is a cluster that is configured to use a VPC from a project different from where the cluster is being deployed. For more information, see Google's documentation on link:https://cloud.google.com/vpc/docs/shared-vpc#shared_vpc_networks[shared VPC networks].

> A shared Virtual Public Cloud (VPC) enables an organization to connect resources from multiple projects to a common VPC network. You can communicate within the organization securely and efficiently using internal IPs from that network. For more information about VPC, see Google's documentation on link:https://cloud.google.com/vpc/docs/shared-vpc[Shared VPC overview].

Reviewer 2: ```suggestion
A shared VPC enables an organization to connect resources from multiple projects to a common VPC network. You can communicate within the organization securely and efficiently by using internal IPs from that network. For more information about VPC, see link:https://cloud.google.com/vpc/docs/shared-vpc[Shared VPC overview] in the GCP documentation.
```
VPC was expanded in the previous paragraph, so you don't need to define it again.

> ! using _noun_
To avoid ambiguity, replace this gerund with either "by using" or "that uses".

And "in the GCP documentation" is used many places in the docs, where "Google's documentation on" is only used in the preceding paragraph. If you're up for a mildly out-of-scope change, please change both instances in this file.

Author: Fixed :)
